### ISSUE TITLE
Website improvements - installation and code editor

### ISSUE THREAD

**Issue by adorayi** [cds-snc/design-gc-conception#862]
- [x] Slots in code preview of header (and others?) show an empty input field with no hints
- [x] Explore options for the copy code button, users are never going to need to copy both react and angular, vue options
- [x] When we show the code of the components in the code editor sections, it notes Angular & Vue together and then React. Is it possible to include HTML in web components as well to help bring some clarity?
- [x] Also, can we change the wording for this No framework (without node) to just reference HTML?
<img width="889" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK09QQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--61121a4aab9f0fb580c00f6acea017212cdad007/Screenshot%202024-04-22%20at%202.48.49%E2%80%AFPM.png" alt="Screenshot 2024-04-22 at 2.48.49 PM.png" />
<img width="615" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK1NQQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--62ec02eadf263c2d9fc7c3bed54fb6c15a469dae/Screenshot%202024-04-22%20at%202.50.33%E2%80%AFPM.png" alt="Screenshot 2024-04-22 at 2.50.33 PM.png" />

**Comment by melaniebmn** [comment]
@adorayi I added custom copy code buttons to our Storybook code preview - one for the web component code and one for the React component code.

I looked into separating the web components and React code preview into two. It would involve having 2 separate iframes with two separate property tables for which we would have to display tabs (or something similar) on the live demo section on our code page. We could look into doing that when we redesign the site, but it seems out of scope as a quick fix since it would require writing new stories for all components, design input, etc.
